docs/sphinx/installation/InstallWindows.rst

```diff
@@ -140,3 +140,157 @@ When finished running CHIPSEC stop/delete service:
 
    ``sc stop chipsec``
    ``sc delete chipsec``
+
+Windows PCI Filter Driver
+-------------------------
+
+**Filter driver background**
+
+Since July 31, Microsoft has released Windows 2020-KB4568831 (OS Build 19041.423) Preview. you should not access the PCI configuration space using the legacy API, it might result in the Windows BSOD (Blue Screen of Death). The BSOD trigger condition is “Windows version >= (OS Build 19041.423) && Secure Devices (SDEV) ACPI table && VBS enabled”. Therefore, we have introduced a PCI filter driver that collaborates with the chipsec main driver to access the PCI configuration space. If you do not meet these conditions, you do not need to install the PCI filter driver, and you can use the chipsec main helper driver directly.
```

Comment:
```suggestion
Since July 31, 2020 Microsoft has released Windows 2020-KB4568831 (OS Build 19041.423) Preview. Microsoft recommends to not access the PCI configuration space using the legacy API, as it might result in the Windows BSOD (Blue Screen of Death). The BSOD trigger condition is “Windows version >= (OS Build 19041.423) && Secure Devices (SDEV) ACPI table && VBS enabled”. Therefore, CHIPSEC now includes a PCI filter driver which supplements the original CHIPSEC Windows Driver to access the PCI configuration space. A system only requires the PCI Filter Driver if the conditions above are met. 
```

Reply:
done